.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: bug
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Environment Details (include every applicable attribute)
+  * Operating System+version: **Linux Ubuntu 18.04**
```

Comment:
Maybe remove the placeholders for the attributes ?
```suggestion
  * Operating System+version:
```

Reply:
done